styleguide/_themes/derek/scss/components/_tabs.scss

```diff
@@ -71,7 +71,7 @@
 }
 
 @media only screen and (max-width : 992px) {
-  .tab-content{
+  #myTab .tab-content{
```

Comment:
Removed those importants. sorry I was debugging a weird preview thing. @SyntaxStacks and I talked about the selector. He's +1. Up to you @bighappyface on merge.